packages/features/ee/teams/pages/availability.tsx

```diff
@@ -30,9 +30,9 @@ export function TeamAvailabilityPage() {
   });
 
   // prevent unnecessary re-renders due to shell queries
-  const TeamAvailability = useMemo(() => {
-    return <TeamAvailabilityScreen team={team} />;
-  }, [team]);
+  // const TeamAvailability = useMemo(() => {
+  //   return <TeamAvailabilityScreen team={team} />;
+  // }, [team]);
```

Comment:
Why?

Reply:
This page isn't being used in V2 and it was causing type errors